getting-started/python/interaction/requires/README.md

```diff
@@ -0,0 +1,43 @@
+# This directory contains the implementation of RGB component.
+
+After implementing the RGB component. We can create another component that will use the information transmitted by the RGB component.
+
+This folder contains only those files which need to be changed to implement a RGB component i.e. the `CDSL` file, the specificworker.py file in src/ directory, and the config file in etc/ directory.
+
+- If you open the `CSDL` file, you can see `requires` keyword is used, which specifies that the component is required here.
```

Comment:
**requires** reference is to the interface, not for a component.

Reply:
Fixed